build/openresty/patches/lua-cjson-2.1.0.10_01-empty_array.patch

```diff
@@ -1,12 +0,0 @@
-diff -ruN a/bundle/lua-cjson-2.1.0.8/lua_cjson.c b/bundle/lua-cjson-2.1.0.8/lua_cjson.c
---- a/bundle/lua-cjson-2.1.0.10/lua_cjson.c	2022-01-11 15:11:17.495464192 +0800
```

Comment:
This bug has been fixed in https://github.com/openresty/lua-cjson/pull/82, I think that we should remove this patch.

Reply:
@chronolaw that is what the PR does. It removes that patch.